common/src/Assets/Texture.h

```diff
@@ -46,7 +46,9 @@ namespace TrenchBroom {
 
             size_t m_usageCount;
             bool m_overridden;
-            
+
+            GLenum m_format = GL_RGB;
```

Comment:
Yes, I used a C++11 feature, did not know that C++11 is disabled by default. Will fix.